examples/common.yaml

```diff
@@ -132,7 +132,7 @@ openstack::swift::hash_suffix: 'pop-bang'
 
 ######## Nova
 
-openstack::nova::libvirt_type: 'kvm'
+openstack::nova::libvirt_type: 'qemu'
```

Comment:
Thanks Colleen for the clarification, I will work on this and resubmit the
patch in a week.

I hope that's an acceptable timeline :)

Moreover, I would like some mentoring so I will seek help on IRC.

regards
/sanjay

On Thu, Jun 11, 2015 at 7:15 AM, Colleen Murphy <notifications@github.com>
wrote:

> In examples/common.yaml
> <https://github.com/puppetlabs/puppetlabs-openstack/pull/199#discussion_r32183988>
> :
>
> > @@ -132,7 +132,7 @@ openstack::swift::hash_suffix: 'pop-bang'
> >
> >  ######## Nova
> >
> > -openstack::nova::libvirt_type: 'kvm'
> > +openstack::nova::libvirt_type: 'qemu'
>
> The puppet-openstack module on stackforge is deprecated.
>
> There is also stackforge/puppet-openstack-cloud but it does not have any
> vagrant examples so this change would not be applicable.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/puppetlabs/puppetlabs-openstack/pull/199/files#r32183988>
> .
>



-- 
Sanjay Upadhyay
http://saneax.blogspot.com